app/src/main/java/uk/gov/onelogin/login/ui/splash/SplashScreenViewModel.kt

```diff
@@ -68,14 +76,15 @@ class SplashScreenViewModel @Inject constructor(
         navigator.navigate(LoginRoutes.AnalyticsOptIn)
     }
 
-    fun retrieveAppInfo() {
+    fun retrieveAppInfo(callback: suspend CoroutineScope.() -> Unit) {
```

Comment:
Could rename `callback` here to make it obvious that's it's an 'onSuccess` callback